src/plugins/ear-training/ear-training.yml

```diff
@@ -10,6 +10,9 @@ solve:
 reset:
   en: Reset
   de: Zurücksetzen
+titleLabel:
+  en: Title
+  de: Titel
```

Comment:
Hint hint, there's a title in `common.yml`

Reply:
Took that one (also in quick-tester), thanks